src/routes/sessions/[id]/+page.svelte

```diff
@@ -47,6 +47,8 @@
 
 	const shouldConfirmDeletion = writable(false);
 
+	$: isServerConnected = !!$settingsStore?.ollamaServer;
+	$: isModelChosen = !!$settingsStore?.ollamaModel;
```

Comment:
> what do you think of closing this PR

@GregoMac1 unless you mean closing it without merging it, this PR it needs additional work.

If `$settingsStore.ollamaServer` is defined that doesn't necessarily mean that `isServerConnected == true`.

`isServerConnected` will be `true` if we are able to get a status 200 when fetching models from `/api/tags`.
Right now that only happens when we call `getModelsList()` in `settings/+page.svelte`.

If you want to limit this PR to disabling the submit button if there are no models and do the rest in another one, that's totally fine.

But in that case I'd also avoid adding a new `isModelChosen` variable, let's just set that directly in the `<ButtonSubmit>` component without adding single-use abstractions; this syntax will still be reactive:
```
disabled={!prompt || !$settingsStore?.ollamaModel}
```

And also, whatever we do here will need a test assertion/s.